### DISCUSSION TITLE
Git branch 전략

### DISCUSSION THREAD

**Discussion by soeunnPark** [Kernel360/E2E2-CATCHLINE#25] [General]
저희 팀은 프로젝트를 Fork해서 작업하지 않기로 했습니다. 이에 따라서 깃 브랜치 전략에 대해 다시 얘기해봤으면 좋겠습니다.

1. Branch 이름 규칙
2. Branch 단위

**Comment by sangwonsheep** [comment]
1. 기능 개발 : feature-member-entity
    수정 : fix-member-entity 와 같이 branch 이름을 사용하면 될 것 같아요!
2. Branch 단위는 entity+repository, service, controller 단위로 크게 나누고 세부적으로는 기능 하나당 하나의 브랜치로 나누면 좋을 것 같아요!

**Reply by soeunnPark** [reply]
좋은 것 같아요 ~! 추가적으로 PR 날리고 작업이 끝난 브랜치는 삭제하면 좋을 것 같아요!
